### DISCUSSION TITLE
Education Ecosystem Accelerator Grant Proposal

### DISCUSSION THREAD

**Discussion by ProjectManagaerEE** [akash-network/community#447] [Governance Proposals]
Name: Michael Garbade

Email Address: drmjg@livecoding.tv

**Overview**
Our goal is to build 50 educational projects associated with Akash to train 350,000 developers from Education Ecosystem. These developers will be able to deploy their dApps on Akash network. 
Projects related to:
Akash Container Platform - deployment platform for hosting and managing
Akash a command-line tool for leasing and deploying apps 
Hosting any containers that can run Cloud-Native application
Deploying a single-tier web application Akash network
Launch an example Minecraft deployment on the Akash Network
Set up an Akash GPU Provider with Praetor

Example projects
https://educationecosystem.com/ivanccrockart/2PjK8-building-a-terrifying-horror-game-unreal-engine
https://educationecosystem.com/s1l3nceer/2bPEr-unity-game-development-artificial-intelligence-multiplayer-and-advance
https://educationecosystem.com/elliottminns/lK6rL-how-to-build-advanced-cryptocurrency-trading-bot-in-nodejs

Goal is to teach the following traditional 350,000 developers how to use tech stack they command and use Akash into their dApps to improve its efficiency, reduce cost and increase resilience:

Web developers:
	Javascript developers
	Python developers
	Ruby developers
	PHP developers
	Java developers

Mobile App developers
	Android developers
	iOS developers
	HTML5 developers

**Organization website** 
https://educationecosystem.com

**Background of Team**
Michael J. Garbade  - CEO 
https://www.linkedin.com/in/garbade/
Ahmed Safadi  - CTO
Muhammad Shoaib  - COO

Michael J. Garbade is a serial entrepreneur, hybrid business executive and Python engineer who enjoys building applications in Python, Django, and Sencha Touch. 

Ahmed Safadi is the CTO and tech lead. He has extensive experience building Django and Python applications for different companies in Europe, China and Turkey prior to Education Ecosystem.

Muhammad Shoaib leads the content creation team at Education Ecosystem (COO). He has eight years of experience working with startups in different roles including operations management, digital marketing, and business development. 

**Deck:**
https://drive.google.com/file/d/1VeCtx9FRe5n1QAdTeDq7Z2bPosoRRWYB/view?usp=sharing

**Funding/Budget/Milestones**
We would like to apply for a $100K USD grant to build these 50 projects. These projects will be made in a span of 3-6 months.

Depending on the size of grant:
Phase 1: 5000 developers
Phase 2 5000 developers
Phase 3: To be decided

Budget Distribution
50%: Devs - Content creation
20%: Audit & Distribution
25%: Marketing

**What are the problems that your project aims to solve?**
The 50 projects that will be created for Akash will equip our 350,000 developers with the knowledge and resources related to Akash network. This will enormously increase Akash community.

**Comment by rodri-r** [comment]
Hello Michael:

With all due respect, this sounds absurd.  

Akash already has tons of documentation, tutorials, guides and content showing developers how to achieve some of the things that you mention and we are creating more almost every day.

There is also Akash Edu platform, which already has some educational projects and they are creating more as I type this.  

Praeator also has excellent documentation about setting up an Akash GPU Provider with Praetor and there are also plenty of templates that almost any developer or person with some tech skills are able to launch.  

The Akash community, Insiders and Vanguards are also constantly creating educational content that helps developers to join Akash and we have the Akash Discord server, with tons of people ready to help new developers and non developers 24/7.

This YouTube channel https://www.youtube.com/@europlots is just one of the many examples of great educational content that we already have.  

According to https://ahrefs.com/traffic-checker

Your website is currently getting 964 organic visitors this month (who knows if these are even real people or if any of those visitors actually join your educational programs) The number of visitors to your website has been going down since August, so it is very likely this number will go down more in February, so not sure where you get that you would reach 350,000 developers, which at the end of your proposal you reduce in phases and to only 10,000 developers.

I am capable of providing educational content, step by step guides, for two of the three example projects that you shared and I will happily create this for free for the Akash commnunity when I have enough time. 

This "educational project"  that you propose - "Launch an example Minecraft deployment on the Akash Network" has been completed by many of us several times and our current Zealy program has a similar task and there are plenty of templates for this too.  

So I don't think this would ever be approved by the community, but thanks for your proposal.

**Reply by ProjectManagaerEE** [reply]
Thanks for the summary, but the statements you made are all factually wrong and shows there are misunderstandings.
 
A.	Important: 
https://ahrefs.com/traffic-checker is not a reliable tool to check traffic of a website at all. Website traffic with this tool can be off over 10,000%.
Any person that has extensive marketing experience will not use it as tool to check web traffic of a site. It is mainly suitable for backlink analysis and tracking backlinks (link building). It is only useful with sites 
that run a lot of Google keyword paid ads and paid rankings and use certain trackers to track their paid ad traffic.
 
B.	The project examples that were shared are just example projects to showcase how a project looks like. It does not mean that the projects that will be built will have the same topic as in the examples
 
C.	Third, in terms of the scope of work we mentioned that we will detail the exact scope after the rough grant range is known. We have campaigns that start from $20k  - $100k. $20k campaigns just go over 1 month and $100k campaigns go over 5-6 months duration         
1)	The 50 projects that will be created will be real products that will be created by experienced developers (average 5 yrs experience) who will spend about 3-4 weeks working on each 
2)	Each project comes with complete recorded videos and source code and takes the developer in average about 35-40 hrs of work to complete. If you multiply this by 50 projects you get about 2000 hours of work. Depending on the budget size, the top 15 programming languages and their frameworks are covered.
3)	30% of the funds is used as airdrop and competition rewards for the learners to motivate them
4)	20% of funds is used for distribution and marketing in third party channels
5)	For the $100k campaign in the last month we host a 2-day hackathon competition where the best projects are awarded prizes (see 3)
 
D.	The campaign budget size determines how many developers in our community will be exposed to the projects. If it is just a $20k campaign, then the number will of course be lower.

**Reply by piber-dev** [reply]
hello @ProjectManagaerEE,

from the example projects mentioned, i don't think the estimated hours are reasonable either without knowing more of the quality you will produce with. for reference, less than 5 hours was required to create [this step-by-step guide](https://medium.com/@piber.dev/creating-a-basic-web-app-in-python-deploying-it-on-akash-6d81b4500d68) including dealing with **a lot** of environment errors plus recording a demo and supporting another person who was creating a video on the same guide. while it is not reasonable to always expect a high efficiency in a company environment, 35-40 hours is 7-8x the numbers of my example which was created at a sub-optimal pace. perhaps you would spend a lot of effort on the videos, this i don't know. for further reference, for the video + article, a total of $500 in rewards was used or 4x less than you are asking per project. you could try to reach out to tyler or hewi in the discord channel if you want a shot of receiving a one-time grant from the community support proposal for getting started with one or a few projects.

**Comment by CaptainSplince** [comment]
Hey Michael - Thanks for the proposal. I have a few questions.

- You mention your goal is to build 50 educational projects associated with Akash. What are the 50 projects? What do they consist of? How are you defining projects?  Just a little confused on this.

- You mention training 350,000 developers. Where are these developers coming from? How will you train them exactly? 

- Your deck says you have over 28,000 creators. Are these creators familiar with Akash already? How do we know the content they create will be professional and technically correct?

- A clear timeline of activity would be useful to review as well. The who, what, when and how of this whole proposal is missing.

- Do you have any case studies of other similar projects to this one? What where the actual outcomes and how where they measured?

Thanks.

**Reply by ProjectManagaerEE** [reply]
Thanks All for the feedback. 

We understand that the $100k budget is too big. Let us look at the $25k and $40k proposals below  and choose one to continue the discussion in detail. Alternatively, we could use the existing Akash tutorials as content to train the developers and then cut the cost of the project creation. Main issue with what Akash has for now is  that they are not complete projects/products. But if team insists to go ahead with that we could do that:

Proposal 1 ($25k) |  
-- | --
Milestone 1: 7 projects | $12k
Airdrop rewards | $2.5k: airdrop rewards
Competition rewards | $2.5k competition reward
Exposure to 2000 developers | $6k ( $3 per developer)
  |  
  |  
Proposal 2 ($40k) |  
Milestone 1: 5 projects | $10k
Milestone 2: 5 projects | $10k
Airdrop rewards | $5k: airdrop rewards
Competition rewards | $5k competition reward
Exposure to 5000 developers | $10k ( $2 per developer)
  |  

A clear timeline of activity would be useful to review as well. The who, what, when and how of this whole proposal is missing.
We will provide this once a proposal package  is selected (decision is made if projects are to be created or existing Akash content is to be used).


You mention training 350,000 developers. Where are these developers coming from? How will you train them exactly?
The projects that will be created will be used to train the developers. Each project will consist of a video that shows the whole process of building the product from scratch, including resources and source code. The developers are our current registered users on Education Ecosystem


Your deck says you have over 28,000 creators. Are these creators familiar with Akash already? How do we know the content they create will be professional and technically correct?

No, the creators are not fully familiar with Akash yet, but they will spend extensive time studying it and prototyping before the projects are created. There will be quality control and technical sync up with the Akash team for each project ( project outline is created, outline is synced up with Akash team, each module is reviewed before the next one is created).

**Reply by CaptainSplince** [reply]
Thanks @ProjectManagaerEE - I still have some confusion on this.

In short, you will use your network of developers to create educational content (videos) to teach developers on how to do X,Y,Z on Akash, correct?

If this is the case, I'd love to see some case studies from previous engagements.

**Reply by ProjectManagaerEE** [reply]
Hi, 
The client projects we have are under NDA.

We are currently working on a huge project right now till March 15.

I think it will be best for Akash team to first decide if you want to proceed using existing Akash tutorials/content or prefer new projects to be created by Education Ecosystem creators.

**Reply by rodri-r** [reply]
I think the existing and upcoming Akash tutorials/content are top quality and meet exactly what we need

**Comment by DappWiz** [comment]
Thanks @ProjectManagaerEE  for presenting this proposal. However, I've noticed several similarities between your proposal and what Omniflix is already implementing at: https://akashedu.omniflix.tv/

Additionally, the proposed funding amount appears to be unreasonable. We would appreciate a detailed breakdown of the funding allocation, specifying hours and other activities, to gain a better understanding of where the funds will be utilized. Thank you.
